docs/application/native/guides/connectivity/usb-host.md

```diff
@@ -237,6 +237,12 @@ usb_host_release_interface(interface);
 dlog_print(DLOG_INFO, LOG_TAG, "%d bytes of response received: %s", bytes_transferred, buffer);
 ```
 
+<a name="async_transfer"></a>
+## Asynchronous Transfer
+
+Asynchronous transfers, including isochronous and control transfers, are explained with examples on [their API page](../../api/mobile/latest/group__CAPI__USB__HOST__ASYNC__MODULE.html).
```

Comment:
To make it more readable and to maintain consistency with the Tizen docs, can we explain what is asynchronous transfer and link the API page. For example,

## Asynchronous Transfer

Asynchronous transfer is...
Asynchronous transfer includes isochronous and control transfers.
For more information, see [Asynchronous IO](../../api/mobile/latest/group__CAPI__USB__HOST__ASYNC__MODULE.html).

Please verify.